dask/dataframe/tests/test_groupby.py

```diff
@@ -1142,100 +1135,6 @@ def test_aggregate_build_agg_args__reuse_of_intermediates():
     assert len(with_mean_finalizers) == len(with_mean_spec)
 
 
-def test_aggregate_dask():
-    dask_holder = collections.namedtuple("dask_holder", ["dask"])
-    get_agg_dask = lambda obj: dask_holder(
-        {
-            k: v
-            for (k, v) in obj.dask.items()
-            # Skip "chunk" tasks, because they include
-            # SubgraphCallable object with non-deterministic
-            # (uuid-based) function names
-            if (k[0].startswith("aggregate") and "-chunk-" not in k[0])
-        }
-    )
-
-    specs = [
-        {"b": {"c": "mean"}, "c": {"a": "max", "b": "min"}},
```

Comment:
`pandas` doesn't know what to do with these specs and from what I can tell, we never assert correctness on the result, so I've removed the tests and would consider this functionality not supported.